### PR TITLE
[feat] Display logged run messages in Run page

### PR DESCRIPTION
**Pull request by VkoHov**
- Add "Run Messages" tab to the Run page
- Display messages as a timeline
- Implement virtualization for timeline component.
- Enable live update for messages

### CODE REVIEW

**Comment by VkoHov** [comment]
Resolves [#2491](https://github.com/aimhubio/aim/issues/2491)

**Review by arsengit — APPROVED** [review]

File: aim/web/ui/src/pages/RunDetail/RunLogRecords/LogRecordItem/LogRecordItem.d.ts [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+   * @type {string}
+   * @example '5e9f1b9b-7c1a-4b5a-8f0c-8c1c1b9b7c1a'
+   */
+  runId: string;
+  /**
+   * The record type
+   * @type {string}
+   * @example 'ERROR'
+   */
+  type: 'ERROR' | 'WARNING' | 'INFO' | 'DEBUG';
```
[the remark this answers is not in the document]
Comment (roubkar) [reply aim/web/ui/src/pages/RunDetail/RunLogRecords/LogRecordItem/LogRecordItem.d.ts:49]: We're using the same values that came from the db

**Review by gorarakelyan — APPROVED** [review]
lgtm

**Review by KaroMourad — APPROVED** [review]

**Review by gorarakelyan — APPROVED** [review]
lgtm

**Pull request merged**